services/object/src/api.js

```diff
@@ -11,6 +11,9 @@ const DOWNLOAD_METHODS = [
   'HTTP:GET',
 ];
 
+const NAME_PATTERN = /^[\x20-\x7e]+$/;
```

Comment:
This made me miss my C days. I had to load up an ASCII table to check it out. It is space, letters, numbers, and all the expected punctuation characters from a US keyboard.

Reply:
It's the same pattern we use for scopes, so i didn't think to comment it.  I'll add a comment and land.